pkg/cli/snapshot.go

```diff
@@ -286,3 +304,32 @@ var cmdSnapshotCreate = &cobra.Command{
 		}
 	},
 }
+
+var cmdSnapshotDelete = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete a snapshot.",
+	Long:  `This command will delete a snapshot of the data.`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		v := getClient()
+
+		snapshotName, err := cmd.Flags().GetString("snapshot")
+		if err != nil {
+			fmt.Printf("Could not retrieve required argument: snapshot. Error: %s\n", err)
+			os.Exit(1)
+		}
+
+		repository, err := cmd.Flags().GetString("repository")
+		if err != nil {
+			fmt.Printf("Could not retrieve required argument: repository. Error: %s\n", err)
+			os.Exit(1)
+		}
+
+		err = v.DeleteSnapshot(repository, snapshotName)
+		if err != nil {
+			fmt.Printf("Error while deleting snapshot. Error: %s\n", err)
+			os.Exit(1)
+		}
+		fmt.Println("Snapshot operation started.")
```

Comment:
Whoops, could you add a more informative output here? 😄

Reply:
Done 👍🏼